src/pages/Details.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router';
+import { getFinancial } from '../redux/CompanyFinancialSlice';
+import CompanyDetails from '../components/FinancialDetails';
+
+const CompanyFinancialDetails = () => {
+  const { financialstats } = useSelector((state) => state.finance);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getFinancial());
+  }, [dispatch]);
+
+  const { id } = useParams();
+  const financialstat = financialstats.find((comp) => comp.id === id);
+
+  return (
+    <>
+      <CompanyDetails
+        key={financialstat?.id}
+        companySymbol={financialstat?.companySymbol}
+        finalLink={financialstat?.finalLink}
+        reportedCurrency={financialstat?.reportedCurrency}
+        netIncome={financialstat?.netIncome}
+        revenue={financialstat?.revenue}
+        calendarYear={financialstat?.calendarYear}
+        costOfRevenue={financialstat?.costOfRevenue}
+        grossProfit={financialstat?.grossProfit}
+      />
+    </>
+  );
+};
+
```

Comment:
- [ ][ optional ] as one of the requirements is to follow the design provided you should follow colors, font-family, font-size and composition, and space between elements.
- Screenshot of the Requirements 
<img width="497" alt="Screenshot 2023-03-23 at 13 27 56" src="https://user-images.githubusercontent.com/61790587/227219696-42f5232e-7471-4ddc-bdcc-d3316ee068b4.png">

- Font provided with colors 
<img width="1137" alt="Screenshot 2023-03-23 at 13 30 15" src="https://user-images.githubusercontent.com/61790587/227219743-0d12c6eb-b9d5-4451-8de5-ef49e9fac958.png">

<img width="498" alt="Screenshot 2023-03-23 at 12 00 32" src="https://user-images.githubusercontent.com/61790587/227200028-c1ab6779-e746-4065-a35d-c328d6734df8.png">


- [ ] The Details  page should be the same as the one in the design by styling the background, font, colors , the data elements 
- yours 
<img width="398" alt="Screenshot 2023-03-23 at 13 30 37" src="https://user-images.githubusercontent.com/61790587/227219896-22a0b6d6-e5c1-46cb-872c-1cdd024451f1.png">

- Design 
<img width="207" alt="Screenshot 2023-03-23 at 13 29 56" src="https://user-images.githubusercontent.com/61790587/227219940-7274e0fb-cffd-466a-ae54-9481bcc48eba.png">